docs/src/components/Markdown.js

```diff
@@ -28,27 +29,65 @@ const stripIndent = (str: string): string => {
   return str.replace(re, '');
 };
 
-export default function Markdown({ text }: Props): Node {
-  const renderer = new Renderer();
+const makePullRequestLink = listitem => {
+  const pullRequestNumRegex = /\(#\d*\)/g;
+  let pullRequestNum = listitem.match(pullRequestNumRegex);
+  if (pullRequestNum && pullRequestNum.length > 0) {
+    pullRequestNum = pullRequestNum[0].replace(/\D/g, '');
+    const pullRequestURL = `https://github.com/pinterest/gestalt/pull/${pullRequestNum}`;
+    const pullRequestAnchor = `(<a href=${pullRequestURL}>#${pullRequestNum}</a>)`;
+    const listItemWithLink = listitem.replace(
+      pullRequestNumRegex,
+      pullRequestAnchor
+    );
+    return listItemWithLink;
+  }
+  return listitem;
+};
+
+const formatCmpName = listitem => {
+  const cmpNamesToIgnore = ['doc', 'docs', 'internal', 'codemod'];
+  const nameAndUpdate = listitem.split(':', 2);
+  const cmpName = nameAndUpdate[0];
+  if (
+    !cmpNamesToIgnore.includes(cmpName.toLowerCase()) &&
+    nameAndUpdate.length > 1
+  ) {
+    const cmpURL = `https://gestalt.netlify.app/${cmpName}`;
+    const formattedListItem = `<a href="${cmpURL}">${cmpName}:</a>${nameAndUpdate[1]}`;
```

Comment:
@ayeshakmaz could we keep the `:` out of the `<a />` so that's not clickable?

![image](https://user-images.githubusercontent.com/127199/96320502-63ab2080-0fc7-11eb-8992-b58a158e6a6b.png)